marimo/_server/api/endpoints/execution.py

```diff
@@ -108,10 +108,21 @@ async def run_cell(
     app_state.require_current_session().put_control_request(
         body.as_execution_request()
     )
+    print("Why does this not print?")
```

Comment:
Just wanted some input here. I was trying to figure out how the backend works. I was running the backend locally with `marimo edit --headles --no-token`

I feel confident it was running because if I killed the server the frontend gave a warning. Probably a silly question, but not sure why this didn't print when I ran cells.

Reply:
`/run` is hit when a cell executes, not just with the server up.
Does that explain the issue? Not sure otherwise, wasn't able to repro locally

Reply:
On server up, `instantiate` is hit.

When you click the run button for an individual cell, `/run` is hit (and your message should print, at least it does for me).

For the backend, we can add a new endpoint — something like `run_scratchpad` — that just takes the Python code to run, since the code won't be registered as a cell/node in the kernel's graph.

Reply:
yes, sorry, was running a cell and still no print. I'll try it again tomorrow...


I have a `run_scratchpad` in this PR in the same file. That is what you mean right?